tests/backup/backup_basic_test.go

```diff
@@ -59,22 +61,55 @@ func TestBasic(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Backup", specReporters)
 }
 
-var dash *aetosutil.Dashboard
-var _ = BeforeSuite(func() {
-	dash = Inst().Dash
-	log.Infof("Init instance")
-	InitInstance()
+// BackupInitInstance initialises instances required for backup
+func BackupInitInstance() {
+	var err error
+	var token string
+	log.Info("Inside BackupInitInstance")
+	err = Inst().S.Init(scheduler.InitOptions{
+		SpecDir:            Inst().SpecDir,
+		VolDriverName:      Inst().V.String(),
+		StorageProvisioner: Inst().Provisioner,
+		NodeDriverName:     Inst().N.String(),
+	})
+
+	log.FailOnError(err, "Error occurred while Scheduler Driver Initialization")
+	err = Inst().N.Init(node.InitOptions{
+		SpecDir: Inst().SpecDir,
+	})
+	err = Inst().V.Init(Inst().S.String(), Inst().N.String(), token, Inst().Provisioner, Inst().CsiGenericDriverConfigMap)
+	log.FailOnError(err, "Error occurred while Volume Driver Initialization")
+
+	if Inst().Backup != nil {
+		err = Inst().Backup.Init(Inst().S.String(), Inst().N.String(), Inst().V.String(), token)
+		log.FailOnError(err, "Error occurred while Backup Driver Initialization")
+	}
+	// Getting Px version info
+	pxVersion, err := Inst().V.GetDriverVersion()
+	log.FailOnError(err, "Error occurred while getting PX version")
+	commitID := strings.Split(pxVersion, "-")[1]
+	t := Inst().Dash.TestSet
+	t.CommitID = commitID
+	if pxVersion != "" {
+		t.Tags["px-version"] = pxVersion
```

Comment:
Any specific reason for maintaining pxVersion?

Reply:
We need this so that we can keep a track of the px-version we are running on